src/Enum.php

```diff
@@ -102,7 +104,8 @@ public static function checkValue(
 
 			throw new InvalidTypeException(
 				\sprintf(
-					'[%s] is not a valid value for %s, accepted values: %s',
+					'%s [%s] is not a valid value for %s, accepted values: %s',
+					\is_object($value) ? \get_class($value) : $value,
```

Comment:
https://3v4l.org/uLp1A

is_object vraci na pole false, takze by to zase hazelu tu exceptionu.. ja si myslim ze se budou muset udelat dve ruzny hlasky, jedna na scalar a na ten zbytek

Reply:
@aleswita jasně v tom případě bych použil Json::encode... toto mi přijde nejjednoduší řešení s nejméně kódu...